### PR TITLE
Adds display-name annotation

### PR DESCRIPTION
**Pull request by cbron**
https://github.com/rancher/rancher/issues/29934

### CODE REVIEW

**Review by deniseschannon — APPROVED** [review]

**Comment by vincent99** [comment]
The annotations look good, but are the patch directories supposed to be changing in some of them?

File: packages/rancher-backup/charts/Chart.yaml [hunk 1]
```diff
@@ -14,3 +14,4 @@ annotations:
   catalog.cattle.io/ui-component: rancher-backup
   catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
   catalog.cattle.io/scope: management
+  catalog.cattle.io/display-name: "Rancher Backups"
```
Comment (aiyengar2) [inline packages/rancher-backup/charts/Chart.yaml:17]: nit: Should this just be `Backups`? Or should `Monitoring` be `Rancher Monitoring` as well?

Comment (cbron) [reply packages/rancher-backup/charts/Chart.yaml:17]: We decided to just remove all the "Rancher" parts. Backup I will ask.

Comment (cbron) [reply packages/rancher-backup/charts/Chart.yaml:17]: We'll do "backups"

**Pull request merged**